Interrupter Software/drivers/includes/uartdvrinc.h

```diff
@@ -0,0 +1,19 @@
+#ifndef UARTDVRINC_H_
+#define UARTDVRINC_H_
+
+#define BAUD 9600 
+#define F_CPU 16000000UL
+
+#include <avr/io.h>
+#include <util/setbaud.h>
+#include <avr/sfr_defs.h>
+
+void UART_enable(void);
+void UART_disable(void);
+
+void UART_double_speed(int);
```

Comment:
Why you don't use this function?

Reply:
The "void UART_double_speed(int);" is used to double the speed of communication between the MCU and other device. In this code this function is not used because its not necessary. But this is a driver written to control all the features of the UART peripheral. That way the driver becomes universal and can be used somewhere else.